Homebrew/Cask.md

```diff
@@ -1,16 +1,32 @@
-# Homebrew Cask
-[Homebrew-Cask](https://caskroom.github.io/) extends Homebrew and allows you to install large binary files via a command-line tool. Examples of these files is Google Chrome, Dropbox, VLC and Spectacle.
+# Homebrew-Cask
+
+[Homebrew-Cask](https://caskroom.github.io/) extends Homebrew and allows you to
+install large binary files via a command-line tool. You can for example install
+applications like _Google Chrome_, _Dropbox_, _VLC_ and _Spectacle_. No more
```

Comment:
I think it flows better when reading it - they stick out to your eyes and that's the purpose. But since we're not using it anywhere else I guess we don't have to use it here.